lms/djangoapps/edraak_certificates/edraakcertificate.py

```diff
@@ -54,6 +54,8 @@ def get_organization_logo(organization):
         return 'pmijo.jpg'
     elif organization == 'qou':
         return 'qou.png'
+    elif organization == 'MBRCGI':
```

Comment:
Please write the organization name in the lower case. This is needed due to `organization = organization.lower()` line above.